go/libraries/doltcore/schema/typeinfo/typeinfo.go

```diff
@@ -73,6 +73,9 @@ type TypeInfo interface {
 	// the given type.
 	ConvertNomsValueToValue(v types.Value) (interface{}, error)
 
+	// ReadFrom reads a go value from a noms types.CodecReader directly
+	ReadFrom(_ *types.NomsBinFormat, reader types.CodecReader) (interface{}, error)
```

Comment:
Just a nitpick, but since all of the other interfaces have a name for the variable, I'd say this one should too for the format.